src/Application/VerifierControlTrait.php

```diff
@@ -43,8 +43,6 @@ public function redirectVerified($code, $destination = null, $parameters = []):
             $parameters = is_array($destination) ? $destination : array_slice(func_get_args(), 1);
             $destination = $code;
             $code = null;
-        } elseif (!is_array($parameters)) {
-            $parameters = array_slice(func_get_args(), 2);
```

Comment:
Pretty sure you can't remove this.

Reply:
is_array on array returns always true so i thought that it will never pass through and deleted it

Reply:
Yeah it's not really always an array. You can also call it as variadic function.

Reply:
You can pass destination into first parameter. It is magic from nette which is too much complicated for tools like phpstan

Reply:
ah ok, thank you